homeassistant/helpers/entity_platform.py

```diff
@@ -844,13 +846,20 @@ async def _update_entity_states(self, now: datetime) -> None:
             return
 
         async with self._process_updates:
-            tasks: list[Coroutine[Any, Any, None]] = []
-            for entity in self.entities.values():
-                if not entity.should_poll:
-                    continue
-                tasks.append(entity.async_update_ha_state(True))
+            if self._update_in_parallel or len(self.entities) <= 1:
+                # If we know are going to update sequentially, we want to update
```

Comment:
If we know we are going to

Reply:
that we know are going to wait

Reply:
I don't understand this comment and the logic of the code. The variable is called `update_in_parallel` but the comment says `update sequentially`. Those two things are opposites, not equals.

Reply:
I'll rename this to `_update_in_sequence` so its a positive check which will be one less negation

Reply:
https://github.com/home-assistant/core/pull/93693